index.html

```diff
@@ -1222,6 +1316,27 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            User agents SHOULD re-fetch and process the manifest on every page
+            load. When the processing is successful, user agents MAY apply updated
+            manifest to any current and future <a>application contexts</a>
+            associated with the application. When the user agent fetches a
+            manifest as part of an update check, it MUST [=url/equal|compare=] the fetched
```

Comment:
I think this is gives general guidance, but it's still a bit "hand wavy" (which may turn out to be ok, given how reliant this is on browser UI and prompts)... 

We should maybe split this into two parts... just the id member, and then we can tackle updating more specifically.